CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 - Expose `pcommon.NewSliceFromRaw` function (#5679)
 - `loggingexporter`: create the exporter's logger from the service's logger (#5677)
 - Add `otelcol_exporter_queue_capacity` metrics show the collector's exporter queue capacity (#5475)
+- Add `jsoniter` Unmarshaller (#5433)
```

Comment:
This should be more specific.  Where is `jsoniter` used?  Is it added or replacing something?  It looks like this only applies to metrics, is there a reason for that?

Reply:
Compare with jsonpb unmarshaller, jsoniter does not use reflect, more efficient, less GC pressure.

Reply:
>It looks like this only applies to metrics, is there a reason for that?

Traces were done earlier: https://github.com/open-telemetry/opentelemetry-collector/pull/4986